plasmapy/classes/plasma.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, domain_x, domain_y, domain_z, gamma=5/3):
 
         x, y, z = self.x.si.value, self.y.si.value, self.z.si.value
         self.grid = np.meshgrid(x, y, z, indexing='ij') * u.m
+        self.grid = np.squeeze(self.grid)
```

Comment:
So this would turn a grid of shape (1, 1, 5) into (5,), right? We should remember to make sure this won't break things like 1D simulations. I'm not saying it will, just saying it looks like it might.